src/assets/i18n/it.po

```diff
@@ -9225,87 +9240,105 @@ msgid ""
 "time frame to <i>Hourly</i> will  only replicate snapshots that were "
 "created at the beginning of each hour."
 msgstr ""
+"Replicare solo le snapshot che corrispondono ad una data di creazione "
+"definita. Per specificare quali snapshot verranno replicate, impostare questa "
+"casella di controllo e definire i tempi di creazione della snapshot che verrà "
```

Comment:
casella di controllo è "checkbox" ? io userei checkbox

Reply:
oggi GH non va... 

Sì, in realtà sarebbe "casella di spunta".
Per me è uguale utilizzare ITA o ENG. 

Meglio ancora: **spuntare** la casella/checkbox al posto di _impostare_

Ancora:
_"definire i tempi di creazione della snapshot che verrà "_ suona malissimo e modificherei con:
_"definire gli **orari** di creazione..."_